### DISCUSSION TITLE
Inquiry Regarding Non-Mandatory Field Clearing with drop down options (Version 1.9)

### DISCUSSION THREAD

**Discussion by pradeep10kumar** [UtrechtUniversity/yoda#345] [Q&A] [answered]
While completing the metadata form, I noticed that once a non-mandatory field (drop down options ) is selected, there seems to be no option to clear or deselect that field.

The complete list of fileds with dropdown options are 
        Related source: Relation type
        Related source: Persistent Identifier: Type
        Creator: Person Identifier: Type
        Contributor: Contributor Type
        Contributor: Affiliation: Affiliation name
        Contributor: Person Identifier: Type

 Is there any method available to clear such fields once they have been chosen?

**Comment by Danny-dK** [comment, marked as answer]
If you press the plus icon next to fields that can have multiple additions, the delete icon becomes available. You can then delete ('clear') the entry and you have an empty metadata field remaining (from the plus action).
